src/components/layout/Header.tsx

```diff
@@ -1,23 +1,135 @@
-import { BarChart3, ChevronDown, Menu, X, Mail } from "lucide-react";
-import { useLocation, matchPath } from "react-router-dom";
+import { BarChart3, Menu, X, Mail } from "lucide-react";
+import { useLocation } from "react-router-dom";
 import { cn } from "@/lib/utils";
 import { useState, useCallback, useEffect } from "react";
 import { useTranslation } from "react-i18next";
-import {
-  Menubar,
-  MenubarContent,
-  MenubarItem,
-  MenubarMenu,
-  MenubarShortcut,
-  MenubarTrigger,
-} from "@/components/ui/menubar";
 import { NewsletterPopover } from "../NewsletterPopover";
 import { useLanguage } from "../LanguageProvider";
 import { HeaderSearchButton } from "../search/HeaderSearchButton";
-import { useScreenSize } from "@/hooks/useScreenSize";
 import useHeaderTitle from "@/hooks/useHeaderTitle";
 import { useAuth } from "@/contexts/AuthContext";
-import { LocalizedLink } from "../LocalizedLink";
+import { LocalizedLink, localizedPath } from "../LocalizedLink";
+import {
+  NavigationMenu,
+  NavigationMenuContent,
+  NavigationMenuItem,
+  NavigationMenuLink,
+  NavigationMenuList,
+  NavigationMenuTrigger,
+} from "../ui/navigation-menu";
+
+interface NavSubLink {
+  label: string;
+  path: string;
+  shortcut?: string;
+}
+
+interface NavLink {
+  label: string;
+  icon?: JSX.Element;
+  path: string;
+  sublinks?: NavSubLink[];
+}
+
+const NAV_LINKS: NavLink[] = [
```

Comment:
Extracted these two from the main component to make that a bit more readable. As a result of that I switched to doing the translation at render and define these with the key instead.